src/app/problems/problem-view/problem-view.component.html

```diff
@@ -14,6 +15,18 @@
         <td>{{uqj?.question?.token_value || 'No Token Value'}}</td>
         <td>{{uqj?.status || 'No Status'}}</td>
         <td>{{uqj?.question?.max_submission_allowed || 'No Value'}}</td>
+        <td>
+            <ng-container>
+                <button type="button" mat-icon-button
+                        [ngClass]="{'red-icon':favoriteStatus[uqj?.question.id], 'gray-icon':!favoriteStatus[uqj?.question.id]}"
+                        (click)="switchFavorite(uqj?.id, favoriteStatus[uqj.question.id], uqj?.question.id)">
+                    <mat-icon>favorite</mat-icon>
+                </button>
+            </ng-container>
+            <ng-container>
+                {{countFavorite[uqj?.question.id]}}
```

Comment:
is there a reason for this to be here?

Reply:
This is just to display the number of favorites besides the favorite button